src/DynamoCore/Search/SearchElements/NodeModelSearchElement.cs

```diff
@@ -12,19 +12,25 @@ namespace Dynamo.Search.SearchElements
     public class NodeModelSearchElement : NodeModelSearchElementBase
     {
         private readonly Func<NodeModel> constructor;
+        private readonly string creationName;
 
+        public override string CreationName { get {return creationName;} }
+        
```

Comment:
@mjkkirschner where are these other changes going to be that you are referring to? Let me know if this change is ok or otherwise I'll think of another solution.